src/Networking/NexusMods.Networking.Downloaders/Interfaces/IDownloadService.cs

```diff
@@ -0,0 +1,95 @@
+using NexusMods.DataModel.Loadouts;
+using NexusMods.DataModel.RateLimiting;
+using NexusMods.Networking.NexusWebApi.Types;
+using NexusMods.Paths;
+
+namespace NexusMods.Networking.Downloaders.Interfaces;
+
+/// <summary>
+/// This is a 'download manager' of sorts.
+/// This service contains all of the downloads which have begun, or have already started.
+/// </summary>
+public interface IDownloadService
+{
+    /// <summary>
+    /// Contains all downloads managed by the application.
+    /// </summary>
+    List<IDownloadTask> Downloads { get; }
+
+    /// <summary>
+    /// This gets fired whenever a download-and-install task is started.
+    /// </summary>
+    IObservable<IDownloadTask> StartedTasks { get; }
```

Comment:
Likewise these can be ReadOnlyObservableCollections and/or `IObservable<IChangeSet<..>>`